celer/group_fast.pyx

```diff
@@ -243,7 +243,7 @@ cpdef celer_grp(
             tmp = alpha / scal
             fscal(&n_samples, &tmp, &theta[0], &inc)
 
-        d_obj = dual(pb, n_samples, norm_y2, &theta[0], &y[0])
+        d_obj = dual(pb, n_samples, alpha, 1.0, norm_y2, 0., &theta[0], &y[0])
```

Comment:
avoid literal and assign a variable `l1_ratio` equal to 1 at the top of this function instead